builder/wasmbuiltins.go

```diff
@@ -0,0 +1,66 @@
+package builder
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/tinygo-org/tinygo/goenv"
+)
+
+var WasmBuiltins = Library{
+	name: "wasmbuiltins",
+	makeHeaders: func(target, includeDir string) error {
+		os.Mkdir(includeDir+"/bits", 0o777)
```

Comment:
Suggestion:
```go
if err := os.MkdirAll(includeDir+"/bits", os.ModePerm); err != nil {
    return err
}
```

Reply:
I'd prefer to keep it as-is:
* The directory should not exist (and the parent should exist), so if that's not the case something is going wrong and I'd like to get an error (which `os.Mkdir` gives).
* `0o777` is a bit more explicit, according to the documentation of `os.ModePerm` it could change in the future while I specifically want these bits.

Reply:
You are not checking the error from `os.Mkdir`

Reply:
...oh, right, you are correct.

Reply:
@aykevl will you modify this PR to check if an error is returned here?

Reply:
I added in another commit in this PR so we can keep this moving along.